### ISSUE TITLE
[D3D9 x32] Resident Evil: Operation Raccoon City crashes before getting to the title screen

### ISSUE THREAD

**Issue by Fumikage141** [doitsujin/dxvk#4172]
The game does not get past the initial logos at all when using DXVK, hard crashing right before the NVIDIA logo is displayed.

The logos/title screen in question:

https://github.com/user-attachments/assets/4f3a3fe7-da05-4d1e-9b33-57689d359796

### Software information
Game: Resident Evil: Operation Raccoon City
Platform: Steam
Windowed Mode

### System information
- OS: Windows 11 Home
- GPU: AMD Radeon RX 6750 XT 
- Driver: AMD Adrenalin 24.7.1 (latest)
- Wine version: N/A
- DXVK version: 2.4

### Apitrace file
[RaccoonCity.trace](https://mega.nz/file/L2BBXDaa#agc2OQs1ivKvLnDUpu8_Y40gfUlmQelHUyTTGAjlQvg), goes past the title screen and into the main menu before exiting the game from there

### Log files
[RaccoonCity_d3d9.log](https://mega.nz/file/33JEGTiY#MIXHE3wsnnER3xVBFjp_pydhUPE2bXgSsb1PnvihcAQ)

**Comment by Blisto91** [comment]
Hi there and thank you for the well made issue.
Sadly it doesn't reproduce when replaying the apitrace but i can reproduce in the game itself. Can you verify that the crashing stops if you make a file called `dxvk.conf` next to the games exe and insert `d3d9.textureMemory = 0`

**Comment by Blisto91** [comment]
Gone ahead and also made a trace where it crashes with dxvk along with a native https://drive.proton.me/urls/P4HP3N30QC#BlojWiNmNiQT

**Comment by Fumikage141** [comment]
> Can you verify that the crashing stops if you make a file called `dxvk.conf` next to the games exe and insert `d3d9.textureMemory = 0`

Yes, I'm able to actually load in and play with that setting. Thanks, wonder why it broke with the default 100 mb

**Comment by K0bin** [comment]
It's probably still broken but disabling managed texture unmapping probably makes it work despite being use-after-free.

I'll take a look this week.

**Comment by K0bin** [comment]
Unfortunately I cannot reproduce the crash using the apitraces. On top of thjat the game can no longer be purchased.

**Comment by Blisto91** [comment]
Workaround have been merged. Issues is kept open in case a dev a have a chance of looking at the root cause at some point.

**Comment by mrdeathjr28** [comment]
in my case work, pass nvidia screen without error

![rca](https://github.com/user-attachments/assets/e49c3b40-2016-4c67-9a93-5178575ca271)

![rcb](https://github.com/user-attachments/assets/cd8a0f8b-7ca3-4bf6-a78a-231aa356a1a3)
